db/test/versions/0023_test.js

```diff
@@ -0,0 +1,67 @@
+const _ = require('lodash');
+const helper = require('../helper');
+const testing = require('taskcluster-lib-testing');
+const Entity = require('taskcluster-lib-entities');
+
+const THIS_VERSION = parseInt(/.*\/0*(\d+)_test\.js/.exec(__filename)[1]);
+const PREV_VERSION = THIS_VERSION - 1;
+
+// (adapted from services/web-server/src/data/GithubAccessToken.js)
+const GithubAccessToken = Entity.configure({
+  version: 1,
+  partitionKey: Entity.keys.ConstantKey('githubAccessTokens'),
+  rowKey: Entity.keys.StringKey('userId'),
+  signEntities: true,
+  properties: {
+    // A unique user ID assigned to the user
+    userId: Entity.types.String,
+    // The Github Oauth access token
+    accessToken: Entity.types.EncryptedText,
+  },
+});
+
+suite(testing.suiteName(), function() {
```

Comment:
This begins to fail after the `.only` is removed, partly due to the absence of a cryptoKey.  But it will also fail because the data's being dropped on upgrade/downgrade.  I bet that could be worked-around by adding a `data-migration` flag to `notImplemented` and just omitting those tests when it's present.

Reply:
Thanks for the tips.

> This begins to fail after the `.only` is removed, partly due to the absence of a cryptoKey.

I see that we set a test crypto key for web server tests [here](https://github.com/taskcluster/taskcluster/blob/426506f034750f7930fdd63e166aef4811889f63/services/web-server/config.yml#L118) - I'm thinking about setting one for the db tests too. Does that seem reasonable to you?